src/tools/pbs_python.c

```diff
@@ -328,6 +328,12 @@ int	   actmode;
 }
 
 int
+action_sched_port(attribute *pattr, void *pobj, int actmode)
+{
+	return 0;
+}
```

Comment:
Thanks for catching this. We added stubs for all the remaining action functions.